### ISSUE TITLE
Grid lines are invisible in Config panel

### ISSUE THREAD

**Issue by Eldaw** [mRemoteNG/mRemoteNG#1798]
The grid lines in the Config panel, by default, are the same colour as the background, making it very difficult to see where to click to enter each field.

![image](https://user-images.githubusercontent.com/3789849/85200505-dba25c00-b2ef-11ea-85d7-84f1513ca543.png)

**Comment by farosch** [comment]
The config panel does not show gridlines at all.
All it shows is a horizontal line whenever a new configuration block begins.
As soon as you click into an editable field the associated property field gets highlighted.
I think this makes it very obvious where to click or what propertie you are about to change.

I will not add gridlines to the config panel, as it has always been like that and you are the first one I have ever heard complaining about it.
Chances are there's a lot more people that like it the way it is now instead of the way you want it to be.

**Comment by Eldaw** [comment]
I think it's because English is not your native language, but your tone comes across as quite rude.

In any case, when you say "The config panel does not show gridlines at all", that is incorrect.

To demonstrate, if I simply change to a dark theme, it can be clearly seen that the config panel actually does render a grid.

![image](https://user-images.githubusercontent.com/3789849/88552375-ee5d2e80-d01b-11ea-8a93-a2c2ff5c0c8f.png)

In my original post, I was quite correct in saying that the problem with the other theme is that the grid is the same colour as the background, which is a usability issue when someone is trying to click on the right side into a specific cell in the grid.

**Comment by Eldaw** [comment]
@sparerd @kmscode May I bring your attention to this report, please? Your colleague is a bit hostile, unfortunately, and has made factually incorrect assertions concerning this issue. I humbly request that it be re-opened.

I had several pleasant email exchanges with some of the previous devs (Riley, Felix, etc.), discussing several areas of functionality in the early days and contributing several important bug reports (as well as making financial donations), and I'm a bit shocked at the unnecessarily hostile response from the current team.

**Comment by farosch** [comment]
> I think it's because English is not your native language, but your tone comes across as quite rude.

Nah, that was absolutely on purpose. I'm sick and tired of people wanting help but not even taking the 10 minutes time to fill out a proper bug report or add some meaningful images!

I would have understood you right away, if you had done so. Reopening now.
